ingestion_tools/dataset_configs/10311_draft.yaml

```diff
@@ -0,0 +1,128 @@
+datasets:
+- sources:
+  - destination_glob:
+      list_glob: '{output_path}/*'
+      match_regex: '/\d*$'
+      name_regex: (.*)
+depositions:
+- metadata:
+    authors: &id001
+      - ORCID: 0000-0003-4685-037X
+        corresponding_author_status: true
+        name: Utz Heinrich Ermel
+        primary_author_status: true
+    dates: &id002
+      deposition_date: '2024-10-15'
+      last_modified_date: '2024-10-15'
+      release_date: '2024-10-15'
+    deposition_description: Standardized tomograms.
+    deposition_identifier: 10311
+    deposition_title: Standardized tomograms
+    deposition_types:
+    - tomogram
+  sources:
+  - literal:
+      value:
+      - 10311
+runs:
+- sources:
+  - destination_glob:
+      list_glob: '{dataset_output_path}/*'
+      match_regex: .*
+      name_regex: (.*)
+    exclude:
+    - Images
+    - dataset_metadata.json
+standardization_config:
+  deposition_id: 10311
+  source_prefix: CZII/standard_tomograms_v2/
+version: 1.1.0
+voxel_spacings:
+- sources:
+  - source_glob:
+      list_glob: '{dataset_name}_{run_name}/*'
+      match_regex: '[\d\.]{5,7}$'
+      name_regex: '([\d\.]{5,7})$'
+    exclude:
+    - logs
+alignments:
+- metadata:
+    alignment_type: LOCAL
+    format: ARETOMO3
+    method_type: projection_matching
+    is_portal_standard: true
+    offset:
+      x: 0
+      y: 0
+      z: 0
+    x_rotation_offset: 0
+    tilt_offset: 0
+  sources:
+    - source_glob:
+        list_glob: '{dataset_name}_{run_name}/*/outputs/aretomo/reconstruct_only/{run_name}.aln'
```

Comment:
Updates the list_glob from `{dataset_name}_{run_name}/{voxel_spacing_name}/outputs/aretomo/reconstruct_only/{run_name}.aln'` to `{dataset_name}_{run_name}/*/outputs/aretomo/reconstruct_only/{run_name}.aln'` as the voxel_spacing is not a parent of the alignment and therefore will not be available at the time of this glob operation.

Confirmed with Utz that we won't have more than one voxel spacing available for a run as the voxel spacing is generated using [this logic](https://github.com/chanzuckerberg/cziimaginginstitute-reconstructor/blob/73fe249c68b753202ef245a81ec0b4f487fc7742/src/reconstructor/core.py). :)